src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Fix it everywhere 
```suggestion
        </label>
        
        <label class="form">
```